src/Place/Pages/PPA002Place/Place.tsx

```diff
@@ -0,0 +1,128 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import React, { useContext, useEffect } from 'react';
+import { FlatList, View, Image, Linking, ScrollView } from 'react-native';
+import { Text, ActivityIndicator } from 'react-native-paper';
+import { Rating } from 'react-native-ratings';
+import BookIcon from 'react-native-vector-icons/Entypo';
+import FetherIcon from 'react-native-vector-icons/Feather';
+import WebIcon from 'react-native-vector-icons/Foundation';
+import TimeIcon from 'react-native-vector-icons/Ionicons';
+
+import { PPA002PlaceController } from './PlaceController';
+import { styles } from './PlaceStyle';
+
+import { BottomTabParamList } from '@/App';
+import i18n from '@/Common/Hooks/i18n-js';
+import { PMC01201GoogleMapPlaceOne } from '@/Place/Models/PDB01MPlace/Contexts/PCT012MPlaceOne/ModelComponents/PMC01201GoogleMapPlaceOne/GoogleMapPlaceOne';
+import { PMC01203PlaceImage } from '@/Place/Models/PDB01MPlace/Contexts/PCT012MPlaceOne/ModelComponents/PMC01203PlaceImage/PlaceImage';
+import { PCT012MPlaceOne } from '@/Place/Models/PDB01MPlace/Contexts/PCT012MPlaceOne/MPlaceOne';
+
+export const PPA002Place = ({ route, navigation }: NativeStackScreenProps<BottomTabParamList, 'PPA002Place'>) => {
+	const { place, setPlaceID, isLoading } = useContext(PCT012MPlaceOne);
+	const { place_id, language } = route.params;
+	const { onCreateItineraryClicked } = PPA002PlaceController(route.params);
+
+	useEffect(() => {
+		setPlaceID(place_id);
+	}, [place_id, setPlaceID]);
+
+	// TODO: https://github.com/Ayato-kosaka/spelieve/issues/321 存在しないplace_idはずっとローディング
+	if (isLoading) {
+		return <ActivityIndicator animating />;
+	}
+	if (!place) {
+		return <Text style={styles.infoText}>{i18n.t('Place Not Found')}</Text>;
+	}
+
+	return (
+		<ScrollView>
+			<View style={styles.container}>
+				<View>
+					<PMC01201GoogleMapPlaceOne />
+					<PMC01203PlaceImage />
+					<Text style={styles.infoText}>{place.name}</Text>
+					<BookIcon name="open-book" size={50} onPress={() => onCreateItineraryClicked(place.name)} />
+					<FetherIcon name="map-pin" size={20}>
+						<Text style={styles.infoText}>{place.formatted_address}</Text>
+					</FetherIcon>
+					<WebIcon name="web" size={20}>
+						<Text
+							style={styles.urlLink}
+							onPress={() => {
+								// eslint-disable-next-line
```

Comment:
eslint-disable-next-line はどのESLintに対するものか明記してほしい
```suggestion
								// eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Reply:
[2e91475](https://github.com/Ayato-kosaka/spelieve/pull/326/commits/2e9147568978db67fe498f98ce247a5373f24b25)